### PR TITLE
feat: v2.2

### PR DESCRIPTION
**Pull request by WoodNeck**
## New Features
- Added new option `initialZoom`
- Added new option `toneMapping`
- Added new option `maxTimeDelta`
- Added new event `loadError`
- Added new event `loadFinish`
- `poster` now receives HTMLElement & CSS selector
- Added helper script `isAvailable`
- LoadingBar now show progress of all loading assets in one loading sequence(init, load)

## ⚠️ Breaking changes
- Reworked whole `shadow` to use contact shadow.

## Bug Fix
- Fixed HDRI can't be drag and dropped in demo playground page.
- Fixed "show skybox" not working in demo playground page.
- Fixed LoadingBar brokes when loading multiple `src`
- Fixed `fixSkinndedBbox` not working for meshopt enabled glTF models

## Demo Page
- https://woodneck.github.io/egjs-view3d/

### CODE REVIEW

File: demo/src/components/View3D.tsx [hunk 1]
```diff
@@ -11,7 +12,7 @@ interface DemoOptions extends Partial<View3DOptions> {
   className: string;
   clickToLoad: boolean;
   showARButton: boolean;
-  showLoadingBar: boolean | string;
+  showLoadingBar: LoadingBarOptions | null;
```
Comment (daybrush) [inline demo/src/components/View3D.tsx:15]: Wouldn't it be dangerous to change showLoadingBar from boolean to object?

Comment (daybrush) [reply demo/src/components/View3D.tsx:15]: It seems that boolean values should also be included.

Comment (WoodNeck) [reply demo/src/components/View3D.tsx:15]: That's the component only for our demo page, so that's fine.

**Review by daybrush — APPROVED** [review]
LGTM

**Review by malangfox — APPROVED** [review]
LGTM

**Pull request merged**